### DISCUSSION TITLE
Feedback for "Daniel’s weekly report November 12, 2021"

### DISCUSSION THREAD

**Discussion by rybak** [bagder/log#4] [General]
> November 12, 2021

Have you considered using ISO 8601 dates in the titles of reports?

> * Put together and shipped curl 7.80.0 and there was much rejoicing. See the blog post for details.

<s>It seems that a link to the mentioned blog post is missing here.</s>

**Comment by rybak** [comment]
> It seems that a link to the mentioned blog post is missing here.

Never mind, found the link later in the report right after starting this discussion.

**Comment by bagder** [comment]
I've stuck to the Month number, year style simply because it feels a little more human and because the date doesn't need to get parsed or sorted by anything.
